api/v2/models/bouquets/bouquets_schema.py

```diff
@@ -5,5 +5,4 @@
 class BouquetsSchema(ma.Schema):
     class Meta:
         model = BouquetsModel
-        fields = ("id", "bouquet_id", "api_key1",  "api_key2"
-                  "auth_credentials", "bouquet_name", "should_refresh")
+        fields = ("id", "bouquet_id", "api_key1",  "api_key2", "auth_credentials", "bouquet_name", "should_refresh")
```

Comment:
This change does nothing but breaks PEP 8 recommendation on maximum line length. https://www.python.org/dev/peps/pep-0008/#maximum-line-length